src/Routes.js

```diff
@@ -137,35 +113,32 @@ export const Routes = (props) => {
                     from={paths.packageDetailSystem.to}
                     to={paths.inventoryDetail.to}
                 />
-                <InsightsRoute
+                <Route
                     path={paths.inventoryDetail.to}
                     component={InventoryPage}
                 />
-                <InsightsRoute exact path={paths.systems.to} component={Systems} />
-                <InsightsRoute
+                <Route exact path={paths.systems.to} component={Systems} />
+                <Route
                     exact
                     path={paths.advisoryDetail.to}
                     component={AdvisoryPage}
                 />
-                <InsightsRoute
+                <Route
                     exact
                     path={paths.advisories.to}
                     component={Advisories}
-                    rootClass="Patchman"
                 />
-                <InsightsRoute
+                <Route
                     exact
                     path={paths.packages.to}
                     component={PackagsPage}
-                    rootClass="Patchman"
                 />
-                <InsightsRoute
+                <Route
                     exact
                     path={paths.register.to}
                     component={RegisterPage}
-                    rootClass="Patchman"
```

Comment:
@karelhala can you check if this className is used in some of the scss files?

Reply:
Yup, checked and I couldn't find it anywhere.